lib/platform.py

```diff
@@ -25,7 +25,13 @@ def load_caps():
     )
   )
   if os.path.exists(capsfile):
-    execfile(capsfile, namespace)
+    global_namespace = {
+      "__file__": capsfile,
+      "__name__": "__main__",
+    }
+    with open(capsfile, 'rb') as file:
+      exec(compile(file.read(), capsfile, 'exec'), global_namespace, namespace)
```

Comment:
Hmmm... any reason why `exec(f.read())` wouldn't work here?

Reply:
yes, exec(f.read()) works, but compile may make it run faster. will do the change